src/website.py

```diff
@@ -63,5 +67,58 @@ def compute_similarity_of_files() -> str:
     return render_template("displaySimilarity.html", text1=texts_start[0], 
                            text2=texts_start[1], text=text)
 
+#navigate to grant call form when button is clicked
+@app.route('/grantcall', methods=['POST', 'GET'])
+def navigateToGrantCallForm() -> str:
+    return render_template("grantCallForm.html")
+
+#calculate similarity and list k most similar scientist upon button click 
+@app.route('/grantcallResult', methods=['POST', 'GET'])
+def calculateGrantCallResult() -> str:
+    #k most similar scientists to display
```

Comment:
```suggestion
    # k most similar scientists to display
```
Space after hash sign. See coding style.